demo_modules/slideshow/load_from_drive_server.ts

```diff
@@ -79,7 +79,15 @@ class DriveItemsDownloader {
               if (paginationToken) {
                 req.pageToken = paginationToken;
               }
-              const response = await driveFilesList(client, req);
+              let response: FileList;
+              try {
+                response = await driveFilesList(client, req);
+              } catch (e) {
+                log.error(
+                  `Unable to download files in folder: ${folderId}: ${e.message}`,
+                );
+                break;
```

Comment:
Just to clarify, what happens if no files load, will the screen go blank?

Reply:
Hm ok eventually we should probably make it revert to default or something. Thanks for fixing this!